modules/rosa-configuring-autorepair.adoc

```diff
@@ -8,12 +8,14 @@
 [id="rosa-configuring-autorepair_{context}"]
 = Configuring machine pool AutoRepair
 
-{hcp-title} supports an automatic repair process for machine pools, called AutoRepair. AutoRepair is useful when you want the ROSA service to detect certain unhealthy nodes, drain the unhealthy nodes, and re-create the nodes. You can disable AutoRepair if the unhealthy nodes should not be replaced, such as in cases where the nodes should be preserved. AutoRepair is enabled by default on machine pools. 
+{product-title} supports an automatic repair process for machine pools, called AutoRepair. AutoRepair is useful when you want the {product-title} service to detect certain unhealthy nodes, drain the unhealthy nodes, and re-create the nodes. You can disable AutoRepair if the unhealthy nodes should not be replaced, such as in cases where the nodes should be preserved. AutoRepair is enabled by default on machine pools.
 
 The AutoRepair process deems a node unhealthy when the state of the node is either `NotReady` or is in an unknown state for predefined amount of time (typically 8 minutes). Whenever two or more nodes become unhealthy simultaneously, the AutoRepair process stops repairing the nodes.
-Similarly, when a new node is created unhealthy even after a predefined amount of time (typically 20 minutes), the service will auto-repair. 
+Similarly, when a new node is created unhealthy even after a predefined amount of time (typically 20 minutes), the service will auto-repair.
 
+ifdef::openshift-rosa-hcp[]
 [NOTE]
 ====
-Machine pool AutoRepair is only available for {hcp-title} clusters. 
+Machine pool AutoRepair is only available for {product-title} clusters.
```

Comment:
Judging by this note, autorepair is only available for HCP clusters so I would condition these 3 modules- rosa-configuring-autorepair.adoc, rosa-autorepair-ocm.adoc, and rosa-autorepair-cli.adoc- out from the Classic docs.

Reply:
Also spotted an incorrect HCP attribute here: 
<img width="857" height="226" alt="image" src="https://github.com/user-attachments/assets/a2f210fe-aa83-44a7-aa29-1e8564786ce3" />